src/server_manager/web_app/location.ts

```diff
@@ -0,0 +1,131 @@
+// Copyright 2021 The Outline Authors
```

Comment:
This should be named `display_location`, `location_formatting`, `location_i18n` or something else indicating this is about i18n.

Reply:
Done